src/main/scala/io/citrine/lolo/trees/regression/RegressionTrainingLeaf.scala

```diff
@@ -31,10 +31,10 @@ class RegressionTrainingLeaf(
   def getFeatureImportance(): scala.collection.mutable.ArraySeq[Double] = {
```

Comment:
I would add on to this comment with the formula used to calculate the impurity now:
sum(L**2 w)/ sum(w) - (sum(L*w))**2/w**2, where L is the regression label, W is the weight.